tests/TestHelpers/GraphHelper.php

```diff
@@ -596,6 +596,34 @@ public static function getMembersList(
 		);
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $adminUser
+	 * @param string $adminPassword
+	 * @param string|null $groupId
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function getAllGroupsAlongWithMembers(
+		string $baseUrl,
+		string $xRequestId,
+		string $adminUser,
+		string $adminPassword,
+		?string $groupId = null
+	): ResponseInterface {
+		// we can expand to get list of members for a single group with groupId and also expand to get all groups with all its members
```

Comment:
but if you list the members of one group, then the function name would be a lie ;-)

Reply:
very true i will give a suitable function name maybe.